# Die Pensionskasse

Die Pensionskasse ist nach dem Kapitaldeckungsverfahren finanziert. Das heisst,
jeder Versicherte der Pensionskassen spart ein individuelles Guthaben an,
aus dem die Rente finanziert wird.

Aktuell (Februar 2024) wird das angesparte Guthaben mit einem gesetzlich
festgeschriebenen Umwandlungssatz von 6.8% (
    [Art. 14 Abs. 2 BVR](https://www.fedlex.admin.ch/eli/cc/1983/797_797_797/de#art_14)
)
in eine Rente umgerechnet. Das bedeutet, das jedes Jahr eine Rente in
der Höhe von 6.8% des bei Erreichen des Rentenalters angesparten
Kaptials ausbezahlt wird.

## Wie lange reicht das angesparte Kapital?

Das scheint eine relativ einfach zu beantwortende Frage zu sein.
Wie oft kann von einem Kapital 6.8% des Anfangskapital weggenommen
werden, bis nichts mehr übrig ist?

In [18]:
counter = 0
savings = 100
pension = 6.8

while savings > 0:
    savings -= pension
    counter += 1

print(f'Die Ersparnisse reichen für {counter} Jahre.')

Die Ersparnisse reichen für 15 Jahre.


So weit so gut. Aber wie lange sollten die Ersparnisse tatsächlich
reichen? Dazu muss auf die statistische Restlebenserwartung bei
Erreichen des Rentenalters abgestellt werden. Diese findet sich in den
sogenannten *Sterbetafeln*. Diese werden nach Geschlechtern getrennt
durch das Bundesamt für Statistik zur Verfügung gestellt 
([Männer](https://www.bfs.admin.ch/bfs/de/home/statistiken/bevoelkerung/geburten-todesfaelle/lebenserwartung.assetdetail.27225580.html),
[Frauen](https://www.bfs.admin.ch/bfs/de/home/statistiken/bevoelkerung/geburten-todesfaelle/lebenserwartung.assetdetail.27225585.html)).

Gemäss diesen Tabellen haben Männer, welche 2022 65 Jahre alt geworden
sind, eine Restlebenserwartung von 19.8 Jahren, Frauen gar eine solche
von 22.5 Jahren. Den Männern fehlen nach obiger Berechnung somit Kapital
für 4.8 Jahre Rente, den Frauen für 7.5 Jahre Rente.

Diese Aussage ist zu relativieren.

Das angesparte Kapital wirft auch nach
erreichen des Rentenalters eine Rendite ab.  

Wie hoch muss die Rendite sein, damit das Kapital für eine Rente bis
ans Lebensende reicht?  
Der Einfachheit halber wird davon ausgegangen, als würden gleich viele
Männer wie Frauen in Rente gehen und daher auf den Mittelwert der
Restlebenserwartung abgestellt.

In [32]:
savings = 100
pension = 6.8
counter = int(19.8 + 22.5) // 2 + 1
remaining_savings = [savings - i * pension for i in range(counter)]
interest = 0.001

while remaining_savings[-1] <= 0:
    for i in range(1, counter):
        remaining_savings[i] = (remaining_savings[i - 1] - pension) * (1 + interest)
    interest += 0.001


print(f'Die erforderliche Verzinsung beträgt {100 * interest:.2f} %.')
for savings in remaining_savings:
    print(f'{savings:.2f}', end=' ')

Die erforderliche Verzinsung beträgt 4.10 %.
100.00 96.93 93.73 90.41 86.95 83.36 79.62 75.74 71.69 67.49 63.12 58.57 53.84 48.92 43.81 38.49 32.96 27.20 21.22 14.99 8.52 1.79 

Damit ist die Frage für die Zukunft beantwortet.

Aber wurde diese Rendite in der Vergangenheit tatsächlich erreicht?

Gemäss einer 
[Zusammenstellung der UBS (aufgerufen im Februar
2024)](https://www.ubs.com/microsites/focus/de/markets/2022/pension-fund-performance.html)
wiesen die Pensionskassen in den Jahren von 2006 bis 2023 im Mittel die
folgenden Renditen auf:

In [20]:
import pandas as pd

In [21]:
actual_performance = pd.read_csv('../data/pk_performance.csv', header=0,
                                 dtype={'Jahr': int, 'Rendite': float})
actual_performance

Jahr  Rendite
0   2006     5.94
1   2007     1.92
2   2008   -12.84
3   2009    10.59
4   2010     2.83
5   2011    -0.34
6   2012     7.16
7   2013     6.05
8   2014     7.36
9   2015     0.86
10  2016     3.54
11  2017     7.96
12  2018    -3.28
13  2019    11.29
14  2020     4.09
15  2021     8.40
16  2022    -9.58
17  2023     4.97

In [23]:
durchschnittliche_rendite = actual_performance['Rendite'].mean()
print(f'Die effektiv erzielte durchschnittliche Rendite gemäss der daten beträgt {durchschnittliche_rendite:.2f} %.')

Die effektiv erzielte durchschnittliche Rendite gemäss der daten beträgt 3.16 %.


In der Folge wird versucht zu zeigen, was das für die Renten bedeutet.
Die Daten der UBS decken den Zeitraum von 2006 bis 2023 ab. Das sind 18
Jahre. Damit Daten für die mittlere Restlebenserwartung zur Verfügung
stehen, werden die Jahre 2002 bis 2005 mit der durchschnittlichen
Rendite gemäss der UBS Datenreihe ergänzt.

In [24]:
pre_years = [[2002, 3.16],
             [2003, 3.16],
             [2004, 3.16],
             [2005, 3.16],]

df = pd.DataFrame(pre_years, columns=['Jahr', 'Rendite'])
df

Jahr  Rendite
0  2002     3.16
1  2003     3.16
2  2004     3.16
3  2005     3.16

In [25]:
extendend_performance = pd.concat([df, actual_performance], ignore_index=True)
extendend_performance

Jahr  Rendite
0   2002     3.16
1   2003     3.16
2   2004     3.16
3   2005     3.16
4   2006     5.94
5   2007     1.92
6   2008   -12.84
7   2009    10.59
8   2010     2.83
9   2011    -0.34
10  2012     7.16
11  2013     6.05
12  2014     7.36
13  2015     0.86
14  2016     3.54
15  2017     7.96
16  2018    -3.28
17  2019    11.29
18  2020     4.09
19  2021     8.40
20  2022    -9.58
21  2023     4.97

In [27]:
performance = extendend_performance['Rendite'].to_list()

available_capital = [100]

for i in range(1, len(performance)):
        available_capital.append((available_capital[i - 1] 
                                  - pension) * 
                                 (1 + performance[i]/100))
    
years = extendend_performance['Jahr'].to_list()

result = pd.DataFrame({'Jahr': years, 'Verfügbares Kapital': available_capital})

result

Jahr  Verfügbares Kapital
0   2002           100.000000
1   2003            96.145120
2   2004            92.168426
3   2005            88.066068
4   2006            86.093272
5   2007            80.815703
6   2008            64.512087
7   2009            63.823797
8   2010            58.637570
9   2011            51.661323
10  2012            48.073393
11  2013            43.770434
12  2014            39.691458
13  2015            33.174324
14  2016            27.307975
15  2017            22.140410
16  2018            14.837245
17  2019             8.944650
18  2020             2.232366
19  2021            -4.951316
20  2022           -10.625539
21  2023           -18.291589

Das Resultat der Simulationsrechnung zeigt, dass statt der erfordlichen
21 Jahre lediglich deren 19 abgedeckt sind.

Der individuelle Fehlbetrag entspricht gut 15%.